src/main/java/dan200/computercraft/shared/turtle/apis/TurtleAPI.java

```diff
@@ -446,6 +449,40 @@ else if( side.equalsIgnoreCase( "right" ) )
                     return new Object[] { null };
                 }
             }
+            case 42:
```

Comment:
It'd probably be nicer to split this into a separate function so one is not duplicating functionality. Maybe do something like [the main `equip*` functions](https://github.com/Wilma456/ComputerCraft-1/blob/f5812c5d29461080fed1d4044939e23e88fd5063/src/main/java/dan200/computercraft/shared/turtle/apis/TurtleAPI.java#L404-L413), and create a separate `TurtleGetEquipCommand` class.